org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,87 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+add-extensions:
+  org.kde.CrowTranslate.tessdata:
```

Comment:
What is this for?

Reply:
It's needed for tesseract models for screen reading. I used [this](https://github.com/flathub/io.github.manisandro.gImageReader/blob/master/io.github.manisandro.gImageReader.yml) app, which also uses Tesseract, as a reference.